src/main/java/nl/tudelft/ti2206/group9/audio/AudioPlayer.java

```diff
@@ -52,15 +52,24 @@ private void initializeTune(final String source) {
 	}
 
 	/**
-	 * Starts the initialized soundtrack.
+	 * Starts the initialized tune.
+	 * If the AudioPlayer has to be checked whether it is running,
+	 * it is checked. If not, then it's not.
+	 * @param checkStopped indicates whether to check the audioPlayer.
 	 */
-	public final void play() {
+	public final void play(final boolean checkStopped) {
 		try {
 			if (State.isSoundEnabled()) {
 				if (audioClip == null) {
 					initializeTune(path);
 				}
-				audioClip.play();
+				if (checkStopped) {
+					if (!this.isRunning()) {
```

Comment:
Could these two if-statments be refactored into one using `and`?

<review, logic